src/css/guide/sidebar.css

```diff
@@ -1,7 +1,11 @@
 .sidebar {
 	margin: 0;
-	padding: 2em 1em 2em 2.5em;
-	border-bottom: 1px solid #eee;
+	padding: 2em 1em 2em 1.5em;
+	position: fixed;
+	width: 14em;
+	height: calc(100% - 4em);
```

Comment:
Could you give a rough fallback value in the line before in case a drowser doesn't understand calc?